src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -471,7 +471,11 @@ def check_config(self):
     def get_config_actions(self) -> list[Action]:
         return [
             (
-                gettext("Configuration"),
+                _("Manage API groups"),
+                reverse("admin:zgw_apis_zgwapigroupconfig_changelist"),
+            ),
+            (
+                gettext("Defaults configuration"),
                 reverse(
                     "admin:zgw_apis_zgwapigroupconfig_changelist",
```

Comment:
These two have the same URL, I guess this should just display `Manage API groups`?

Reply:
d'oh, will update!

Reply:
okay, this was a mistake - the solo model is gone, so there should not be a link anymore to "global/defaults configuration", only to the API groups list. Updated the PR!